### PR TITLE
Fixed bug on hunter edit page where moves no showed

### LINKED ISSUE #205: Cannot add new moves on Hunter Edit page
**Describe the bug**
Hunters cannot add moves they don't already have on the Hunter Edit page.

**To Reproduce**
1. Go to /hunter/:id/edit
2. Only moves the hunter already has are seen

**Expected behavior**
All moves are visible, and moves the hunter has are selected. The hunter can add new moves by selecting the checkbox and hitting submit.

**Screenshots**
image

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Additional context**

### PR DESCRIPTION
**Pull request by muydanny**
## Description of Feature or Issue
closes #205 

:bug::bug::bug::bug:
## User-Facing Changes

Screen Shot 2021-02-02 at 2 46 05 PM

### CODE REVIEW

File: app/models/move.rb [hunk 1]
```diff
@@ -35,11 +35,13 @@ class Move < ApplicationRecord
   has_many :hunters, through: :hunters_moves
 
   scope :not_basic, -> { where.not(type: 'Moves::Basic') }
+  # Filters to only moves the hunter h
```
Comment (ChaelCodes) [inline app/models/move.rb:38]:
```suggestion
  # Filters to only moves the hunter has
```
Dangling h

**Comment by ChaelCodes** [comment]
Hey @muydanny can you merge in the new changes, and update that?

**Review by ChaelCodes — APPROVED** [review]
This looks good!

**Comment by muydanny** [comment]
WOO 🙋🏽‍♂️

**Pull request merged**